_data/ethanelliott/ethanelliott.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Ethan Elliott
+institution: Ontario Tech University
+profile_pic: ethanelliott.jpg
```

Comment:
Incorrect file name, please do one of the following:
1. Change `ethanelliott.jpg` to `ethanelliott.png` in the markdown file.
2. Rename `ethanelliott.png` to `ethanelliott.jpg`.
I'd suggest you make the change in the markdown file!